bin/mpdevil

```diff
@@ -851,7 +851,7 @@ class Client(MPDClient):
 					else:
 						self.emitter.emit("elapsed_changed", 0.0, 0.0)
 				elif key == "bitrate":
-					self.emitter.emit("bitrate", float(val))
+					self.emitter.emit("bitrate", val)
```

Comment:
I think we should test here for 'val == "0"' and return None to avoid passing trough "magic" values.